src/modules/i18n/containers/I18nProvider.js

```diff
@@ -89,13 +90,13 @@ export class I18nProvider extends React.Component<PropsType, StateType> {
     this.i18n.changeLanguage(targetLanguage)
   }
 
-  componentWillMount () {
+  componentDidMount () {
```

Comment:
Hmm... I think we shouldn't do this change to the new react version in this task. Maybe we should create a new task for that - These changes could potentially have unwanted side effects...

Reply:
I tested the changes and there should be no problems. Can change it back to the UNSAFE_ methods though.
I only had to change the lifecycle methods in the I18nProvider. So if there are any problems we know its because of this change.

Reply:
Can change it back but I don't see an issue with doing it on DidMount.
Actually we are not allowed to do it in WillMount. WillMount does not allow side-effects.